docs/guide/user_management/token.md

```diff
@@ -0,0 +1,97 @@
+# Token
+
+[[= product_name_com =]] uses the token system in the registration process to create a double opt-in possibility.
+The token service can generate a unique token which is valid for a given time.
+
+After user registration a token is created and stored in the database.
+When the user clicks the URL that they receive in an email, the token is validated and fetched from the database.
+After the user has been activated, the token is removed from the database.
+
+## Creating a token
+
+A token contains:
+
+- `userId`
+- parameters 
+- the time in which it is valid
+- a service and method which are called if a customer uses the link with the token
+
+``` php
+use Silversolutions\Bundle\ToolsBundle\Entity\Token;
+use Silversolutions\Bundle\ToolsBundle\Services\TokenService;
+
+$newsletterTokenData = new NewsletterTokenData();
+$newsletterTokenData->setParams($newsletterParams);
+$newsletterTokenData->setCustomerProfileData($customerProfileData);
+$actionServiceMethod = 'subscribeNewsletter';
+$actionService = 'siso_newsletter.newsletter.newsletter_service';
+/** @var Token $token */
+$token = $this->tokenService->createToken(
+    $currentUser->id,
+    $newsletterTokenData,
+    $validUntil,
+    $actionService,
+    $actionServiceMethod,
+    Token::TOKEN_TYPE_ONE_TIME
+);
+```
+
+## Using the token
+
+You can use the token in the built-in token controller to call a service and method, or in a custom controller or service.
```

Comment:
```suggestion
You can use the token to call a service and method in the built-in token controller, or in a custom controller or service.
```
Is it 
"... in the built-in token controller that calls a service and method..."
or
"the token to call a service and method in the built-in token controller, or in a custom controller or service"
?